packages/cosmic-swingset/lib/ag-solo/web.js

```diff
@@ -78,20 +85,23 @@ export async function makeHTTPListener(basedir, port, host, rawInboundCommand) {
       return true;
     }
 
-    // Validate the private accessToken.
-    const accessToken = await getAccessToken(port);
-    const reqToken = new URL(`http://localhost${req.url}`).searchParams.get(
-      'accessToken',
-    );
-
-    if (reqToken !== accessToken) {
-      log.error(
-        id,
-        `Invalid access token ${JSON.stringify(
-          reqToken,
-        )}; try running "agoric open"`,
+    // Bypass accessToken just for the wallet bridge.
+    if (req.url !== '/private/wallet-bridge') {
+      // Validate the private accessToken.
+      const accessToken = await getAccessToken(port);
+      const reqToken = new URL(`http://localhost${req.url}`).searchParams.get(
+        'accessToken',
       );
-      return false;
+
+      if (!verifyToken(reqToken, accessToken)) {
+        log.error(
+          id,
+          `Invalid access token ${JSON.stringify(
```

Comment:
minor: why JSON.stringify? from what I can tell `searchParams.get` always returns a string or `null`. (aside: I hope our future eslint rules allow incidental logging like this to be more compact and less distracting from the primary `verifyToken` and `return false` lines)

Reply:
To distinguish between `null` and `"null"`.